.github/workflows/gitleaks.yml

```diff
@@ -0,0 +1,15 @@
+name: gitleaks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  gitleaks:
```

Comment:
Should this have a name to be consistent with the trufflehog job?

Reply:
```suggestion
  gitleaks:
    name: Run gitleaks report
```